deploy-as-code/helm/charts/digit-works/backend/works-inbox-service/values.yaml

```diff
@@ -0,0 +1,73 @@
+# Common Labels
+labels:
+  app: "works-inbox-service"
+  group: "digit-works"
+
+namespace: works
+
+# Ingress Configs
+ingress:
+  namespace: egov
+  enabled: true
+  zuul: true
+  context: "works-inbox-service"
+
+# Container Configs
+image:
+  repository: "works-inbox-service"
+replicas: "1"
+healthChecks:
+  enabled: true
+  livenessProbePath: "/works-inbox-service/health"
+  readinessProbePath: "/works-inbox-service/health"
+appType: "java-spring"
+tracing-enabled: true
+heap: "-Xmx192m -Xms192m"
+java-args: ""
+
+# Additional Container Envs
+env: |
+  - name: SERVER_PORT
+    value: "8080"
+  - name: JAVA_OPTS
+    value: {{ index .Values "heap" | quote }}
+  - name: JAVA_ARGS
+    value: {{ index .Values "java-args" | quote }}
+  {{- if index .Values "tracing-enabled" }}
+  - name: TRACER_OPENTRACING_ENABLED
+    value: "true" 
+  {{- end }}
+  - name: SERVER_CONTEXT_PATH
+    value: /inbox
```

Comment:
Wouldn't it be "/works-inbox-service"? 
Same for the line below.

Reply:
I thought of keeping the same context path as we have in other municipal services. if you want it be "/works-inbox-service", I'll change them.